src/Particular.CodeRules/stylecop.json

```diff
@@ -1,5 +1,9 @@
 ﻿{
   "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
   "settings": {
+    "documentationRules": {
+      "documentInterfaces": false,
```

Comment:
I'm not sure we should set this to false. We should really be documenting public interfaces.

If a given repo has too many violations because of undocumented public interfaces, we can set this to false just for that repo, and add it to our list of outstanding items.

Reply:
Sorry, perhaps I should have explained longer than saying "previous expectations":

By default `documentExposedElements` is `true`. This requires all public types, including interfaces, to be documented.

I confirmed setting `documentInterfaces` to `false` does not override the `documentExposedElements` setting for public interfaces before I submitted this PR.

Reply:
@boblangley OK, I see what you mean now. I had a look at https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md#documentation-requirements and it matches what you are saying.

I find the way these flags work a bit confusing, but ¯`\_(ツ)_/¯`.